src/main/io/ledstrip.c

```diff
@@ -92,7 +92,17 @@ static void ledStripDisable(void);
 
 //#define USE_LED_ANIMATION
 
-#define HZ_TO_MICROS(hz) ((int32_t)((1000 * 1000) / (hz)))
+#define HZ_TO_TIMER_DELAY(hz) ((int32_t)((1000 * 1000) / (hz)))
+
+#define MAX_TIMER_DELAY (5 * 1000 * 1000)
+#define TIMER_DELAY_1_S (1000 * 1000)
+#define TIMER_DELAY_500_MS (500 * 1000)
+#define TIMER_DELAY_400_MS (400 * 1000)
+#define TIMER_DELAY_200_MS (200 * 1000)
+#define TIMER_DELAY_125_MS (125 * 1000)
+#define TIMER_DELAY_100_MS (100 * 1000)
+#define TIMER_DELAY_50_MS (50 * 1000)
+#define TIMER_DELAY_16_MS (16 * 1000)
```

Comment:
@mikeller : Yes. Storing  time in variable (instead of frequency) allows to use constant expressions (`HZ_TO_US(1)`) which are evaluated on compile time. Your implementation stored  frequency and conversion had to be done each time